actions/campaigns.go

```diff
@@ -575,3 +577,79 @@ func GetCampaignBounces(c *gin.Context) {
 		"message": "Campaign bounces not found",
 	})
 }
+
+func PatchCampaignSchedule(c *gin.Context) {
+	campaignID, err := strconv.ParseInt(c.Param("id"), 10, 64)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Id must be an integer",
+		})
+		return
+	}
+	u := middleware.GetUser(c)
+
+	campaign, err := storage.GetCampaign(c, campaignID, u.ID)
+	if err != nil {
+		c.JSON(http.StatusNotFound, gin.H{
+			"message": "Campaign not found",
+		})
+		return
+	}
+
+	body := &params.CampaignSchedule{}
+	if err := c.ShouldBind(body); err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Invalid parameters, please try again",
+		})
+		return
+	}
+
+	if err := validator.Validate(body); err != nil {
+		c.JSON(http.StatusBadRequest, err)
+		return
+	}
+
+	schAt, err := time.Parse("2006-02-01 15:04:05", body.ScheduledAt)
+	if err != nil {
+		c.JSON(http.StatusBadRequest, gin.H{
+			"message": "Invalid parameters, scheduled_at should be format: 2006-02-01 15:04:05",
+		})
+		return
+	}
+
```

Comment:
We'll need to check if the user has permissions to schedule campaigns (boundary check).